compiler/luci/pass/src/QuantizedModelVerifier.test.cpp

```diff
@@ -199,6 +219,45 @@ TEST(QuantizedModelVerifierTest, Logistic_wrong_granularity_NEG)
   }
 }
 
+TEST(QuantizedModelVerifierTest, Softmax)
+{
+  TEST_WITH_GRAPH(SoftmaxTestGraph, Type::U8, Granularity::LayerWise);
+  TEST_WITH_GRAPH(SoftmaxTestGraph, Type::U8, Granularity::ChannelWise);
+  TEST_WITH_GRAPH(SoftmaxTestGraph, Type::S16, Granularity::ChannelWise);
+}
+
+TEST(QuantizedModelVerifierTest, Softmax_wrong_type_NEG)
+{
+  {
```

Comment:
This brace seems needless :)

Reply:
It is for avoiding name aliasing when adding a new test in the same category (testing wrong type).

Without the brackets, we'll have to use different variable names when adding a new test, for example,

```cpp
// test 1
SoftmaxTestGraph g;
..

// test 2
SoftmaxTestGraph g2;
...
```

Reply:
(optional) For this case, IMHO, adding one test for each in test case would be better, that is, 
something like, `Softmax_wrong_type_NEG` -> `Softmax_wrong_type_LW_U8_NEG`
I'll wait for your decision :)

Reply:
Dividing the test case looks good. I've updated the code. PTAL